resources/views/auth/login.html.twig

```diff
@@ -95,6 +95,7 @@
                                 required
                                 type="password"
                                 autocomplete="off" />
+                            <span class="password-caps-warning hidden">Warning: Caps lock engaged</span>
```

Comment:
This entry should be in a lang file with a French equivalent.

Reply:
Damn missed that sorry